assets/scripts/main.js

```diff
@@ -64,7 +64,18 @@ jQuery(document).ready(function($) {
     else {
       $('#reset_related_accessories').hide();
     }
+
+    if (data.isHidden) {
+      // Shows hidden related accessories after product has been added to cart.
+      // This is for cases of AJAX add to cart functionality.
+      jQuery('body').on('added_to_cart', function() {
+        jQuery('html, body').animate({scrollTop: 0}, 400, function () {
+          jQuery('.single-product .related-accessories').slideDown();
+        });
+      });
```

Comment:
This is still the most problematic part from a functional / UX perspective, especially on small mobile screens. 🤔 Just scrolling to the top is a huge assumption about the container being actually in the visible viewport there which might not hold true – and in that case, the theme doesn't have any way to change or opt-out of this behavior.

Can we move the related accessories below the add-to-cart button and slide them into view there?

---
Minor code style issues:
```suggestion
      // Shows hidden related accessories after product has been added to cart via Ajax.
      $('body').on('added_to_cart', function () {
        $('html, body').animate({ scrollTop: 0 }, 400, function () {
          $('.single-product .related-accessories').slideDown();
        });
      });
```